db/schema.rb

```diff
@@ -77,7 +77,7 @@
     t.integer "sluggable_id", null: false
     t.string "sluggable_type", limit: 50
     t.string "scope"
-    t.datetime "created_at", precision: nil
```

Comment:
@zspencer assuming changing precision of this field won't have any impact, ie if we were comparing any date strings that came from this table? Or doesn't matter since db default is ok for this field?  Also why did this change?

Reply:
I presume this is because `precision: nil` is the same as leaving precision off; and between Rails 6 and 7 they made that implicit instead of explicit.